tools/run_tests/performance/README.md

```diff
@@ -104,3 +104,29 @@ Example memory profile of grpc-go server, with `go tools pprof`:
 ```
 $ go tool pprof --text --alloc_space http://localhost:<pprof_port>/debug/heap
 ```
+
+### Configuration environment variables:
+
+* QPS_WORKER_CHANNEL_CONNECT_TIMEOUT
+
+  Consuming process: qps_worker
+
+  Type: integer (number of seconds)
+
+  This can be used to configure the amount of time that benchmark
+  clients wait for channels to the benchmark server to become ready.
+  This is useful in certain benchmark environments in which the
+  server can take a long time to become ready. Note: if setting
+  this to a high value, then the scenario config under test should
+  probably also have a large "warmup_seconds".
+
+* QPS_WORKERS
+
+  Consuming process: qps_json_driver
+
+  Type: comma separated list of host:port
+
+  Set this to a comma separated list of QPS worker processes/machines.
+  Note that the driver will start the "benchmark server" on the first
```

Comment:
I think this is not accurate.  from the list, the first N items will be consumed to represent servers based on the json configuration of how many servers are requested for given scenario. The next K will be taken as clients).
https://github.com/grpc/grpc/blob/158fb288fc9807ddf9d6adca6f3d4803e855ae34/tools/run_tests/performance/scenario_config.py#L137

special cases:
- if you specify num_server + num_clients  less than the number of items in QPS_WORKERS, the rest will get unused.
- i think num_clients=0 used to mean something like "consume all the remaining workers from the list", but I'm not sure if that's the case or not.

It's actually pretty convoluted :-)

Reply:
good catch :) fixed